packages/oui-progress-tracker/_mixins.less

```diff
@@ -7,66 +7,71 @@
     @font-color-active: @oui-progress-tracker-font-color-active,
     @font-color-complete: @oui-progress-tracker-font-color-complete,
     @font-size: @oui-progress-tracker-font-size,
-    @item-max-width: @oui-progress-tracker-item-max-width,
-    @item-padding: @oui-progress-tracker-item-padding,
-    @container-background-color: @oui-progress-tracker-container-background-color,
+    @label-color: @oui-progress-tracker-label-color,
+    @label-color-active: @oui-progress-tracker-label-color-active,
+    @label-color-complete: @oui-progress-tracker-label-color-complete,
+    @label-min-width: @oui-progress-tracker-label-min-width,
+    @label-max-width: @oui-progress-tracker-label-max-width,
+    @background-color: @oui-progress-tracker-background-color,
+    @background-color-active: @oui-progress-tracker-background-color-active,
+    @background-color-complete: @oui-progress-tracker-background-color-complete,
     @container-height: @oui-progress-tracker-container-height,
     @container-margin: @oui-progress-tracker-container-margin
   ) {
     position: relative;
     display: flex;
-    justify-content: space-around;
+    justify-content: space-between;
     width: 100%;
     height: @container-height;
```

Comment:
(optional) It's not from your PR, but all that fixed heights seems a bit off when using dynamic text...
There too much content that overflow the boxes when seen by the development tool.

Reply:
Thanks a lot Alex :). I agree with you. I will try to include these changes too shortly

Reply:
The clean-up is done and there is no height hard-coding. All over-flow has been fixed too :). Thanks for your very valid suggestion :)